fpdf/image_parsing.py

```diff
@@ -353,6 +353,10 @@ def get_img_info(filename, img=None, image_filter="AUTO", dims=None):
     w, h = img.size
     info = {}
 
+    iccp = None
+    if "icc_profile" in img.info:
+        iccp = img.info.get("icc_profile")
```

Comment:
Given that this object is never altered by your code later on,
that means that the `icc_profile` provided by Pillow has the right format for the PDF spec,
no need for any data manipulation? If so that's great and very handy! 😊

Reply:
yes, the ICC profile doesn't need to be parsed or changed

For the validation... I guess I could add something but I'm not sure how useful that might be. img2pdf is usually very thorough in checking everything and it doesn't check this so my guess is that it must be quite uncommon In any case, Pillow depends on [pycms](https://www.cazabon.com/pyCMS/downloads/index.txt) which has a `getProfileInfo`, I guess I could use that

Reply:
do you think it's really necessary? there's a chance that the image won't be read by PDF readers anyway

Reply:
I suggest that you give this a try, as it would be nice to report potential issues regarding ICC profiles to `fpdf2` users,
but I agree that this should relatively uncommon 😊
In fact, my understand is that `Pillow` will raise an error for invalid ICC profiles: https://github.com/python-pillow/Pillow/blob/main/Tests/test_imagecms.py

Reply:
done (I think)